packages/-ember-data/tests/integration/request-state-service-test.ts

```diff
@@ -0,0 +1,230 @@
+import { get } from '@ember/object';
+import { setupTest } from 'ember-qunit';
+import Model from 'ember-data/model';
+import Store from 'ember-data/store';
+import { module, test } from 'qunit';
+import { settled } from '@ember/test-helpers';
+import EmberObject from '@ember/object';
+import { attr, hasMany, belongsTo } from '@ember-data/model';
+
+interface RecordIdentifier {
+  id?: string;
+  type: string;
+  lid: string;
+}
+
+interface JsonApiValidationError {
+  title: string;
+  detail: string;
+  source: {
+    pointer: string;
+  }
+}
+
+class Person extends Model {
+  // TODO fix the typing for naked attrs
+  @attr('string', {})
```

Comment:
ping @mike-north about fixing this, if it's not already fixed. This should be a 5 minute job

Reply:
you can also just do `@attr() name;`